.ci/magic-modules/generate-terraform-all-platforms.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Setup GOPATH
+export GOPATH=${PWD}/go
+# Setup GOBIN
+export GOBIN=${PWD}/dist
+
+set -x
+
+# Create GOBIN folder
+mkdir -p "$GOBIN"
+
+# Create GOPATH structure
+mkdir -p "${GOPATH}/src/github.com/terraform-providers"
+yes | cp -rf "$1" "${GOPATH}/src/github.com/terraform-providers/"
+
+cd "${GOPATH}/src/github.com/terraform-providers/terraform-provider-google"
+
+# Platforms build
+platforms=("linux/386" "linux/amd64" "linux/arm" "linux/arm64" "linux/ppc64" "linux/ppc64le" "linux/mips" "linux/mipsle" "linux/mips64" "linux/mips64le")
+
+for platform in "${platforms[@]}"
+do
```

Comment:
Strongly recommend you use gox instead, as per our email.

Reply:
Done